addons/sourcemod/scripting/shavit-mapchooser.sp

```diff
@@ -45,6 +45,8 @@ int gI_Driver = Driver_unknown;
 Database g_hDatabase;
 char g_cSQLPrefix[32];
 
+bool gB_instantChange = false;
```

Comment:
Is this bool necessary, versus passing `when`?

Reply:
Couldn't figure out how to get this to function using when how u say

Reply:
g_ChangeTime? instead of my bool maybe?